code/drasil-lang/Language/Drasil/Expr.hs

```diff
@@ -4,7 +4,7 @@ module Language.Drasil.Expr where
 
 import Data.Ratio (denominator, numerator)
 import Prelude hiding (sqrt)
-
+import Language.Drasil.Symbol
```

Comment:
This is incorrect, as it adds an unqualified import, when there is a correct precise import below.

Reply:
Changes were made in 3635187475cca3dfcacf1fca2305c36d486209e2.